PodFile

```diff
@@ -1,68 +1,54 @@
 use_frameworks!
 
-def import_base_pods
-    pod 'FMDB', '= 2.6'
-    pod 'CocoaLumberjack', '~> 2.0'
+if ENV["encrypted"]
+    MRDatabaseContentChecker = "MRDatabaseContentChecker/SQLCipher"
+    FMDB = "FMDB/SQLCipher"
+else
+    MRDatabaseContentChecker = "MRDatabaseContentChecker"
+    FMDB = "FMDB"
 end
 
-def import_encrypted_base_pods
-    pod 'FMDB/SQLCipher', '= 2.6'
-    pod 'SQLCipher/fts', '~> 3.1.0'
+abstract_target 'base' do
+    pod FMDB, '= 2.6'
     pod 'CocoaLumberjack', '~> 2.0'
-end
-
-def import_test_pods
-    pod 'Specta'
-    pod 'Expecta'
-    pod 'OCMock'
-    pod 'OHHTTPStubs'
-    pod "MRDatabaseContentChecker", :git => 'https://github.com/rhyshort/MRDatabaseContentChecker.git'
-end
-
-def import_RA_pods
-    pod "Unirest", :git => 'https://github.com/rhyshort/unirest-obj-c.git'
-    pod 'TRVSMonitor'
-    pod 'NSData+Base64'
-end
-
-def import_encrypted_test_pods
-    pod 'Specta'
-    pod 'Expecta'
-    pod 'OCMock'
-    pod 'OHHTTPStubs'
-    pod "MRDatabaseContentChecker/SQLCipher", :git => 'https://github.com/rhyshort/MRDatabaseContentChecker.git'
-end
-
-target :CDTDatastore do
-    platform :ios, '7.0'
-    import_encrypted_base_pods
-end
-
-target :CDTDatastoreTests do
-    platform :ios, '7.0'
-    import_encrypted_base_pods
-    import_encrypted_test_pods
-end
-
-target :CDTDatastoreReplicationAcceptanceTests do
-    platform :ios, '7.0'
-    import_RA_pods
-    import_encrypted_base_pods
-end
-
-target :CDTDatastoreOSX do
-    platform :osx, '10.9'
-    import_encrypted_base_pods
-end
-
-target :CDTDatastoreTestsOSX do
-    platform :osx, '10.9'
-    import_encrypted_base_pods
-    import_encrypted_test_pods
-end
-
-target :CDTDatastoreReplicationAcceptanceTestsOSX do
-    platform :osx, '10.9'
-    import_RA_pods
-    import_encrypted_base_pods
+    pod 'SQLCipher/fts', '~> 3.1.0' if ENV["encrypted"]
+
+    target :CDTDatastore do
+        platform :ios, '7.0'
+    end
+    target :CDTDatastoreOSX do
```

Comment:
Yes we could, but right now the target in the project is still `CDTDatastoreOSX`. I didn't think there was point in changing the name, I'm sure people will still refer to `macOS` as `OSX` for a while anyway.